.claude/audit/scripts/refine.py

```
@@ -0,0 +1,46 @@
+
+def split(ms):
+    methods = [m for m in ms if not m.startswith(("get_", "set_"))]
+    acc = [m for m in ms if m.startswith(("get_", "set_"))]
+    return methods, acc
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--diff", default=".claude/audit/.generated/diff.json")
+    args = ap.parse_args()
+    d = json.load(open(args.diff, encoding="utf-8"))
```

Ruff (0.15.14)

[warning] Use a context manager for opening files

(SIM115)